### ISSUE TITLE
Double-click to open message does nothing

### ISSUE THREAD

**Issue by alcarraz** [thunderbird-conversations/thunderbird-conversations#1890]
* **Thunderbird version:** 91.12.0 (64-bit)
* **Thunderbird Conversation version:** 4.0.22
* **Operating System:** Ubuntu 22.04

**What did you do?** Double Click any email

**What did you expect to happen?** Open in new window as it's set to do in the config

What actually happened? Not a thing

I have re-enabled this extension after a while. I don't remember the behavior before I disabled it last time, but when using thunderbird with the extension disabled, it opens the mail in a new window when I double-click, and when I enable it nothing happens.

This is quite similar to issue #1406, at which I arrived while googling for this issue.

**Comment by Standard8** [comment]
If you open the error console by going to the three bar menu -> Tools -> Developer Tools -> Error Console, does it show anything when you try double clicking?

Could there be another add-on install that is conflicting?

**Comment by alcarraz** [comment]
> If you open the error console by going to the three bar menu -> Tools -> Developer Tools -> Error Console, does it show anything when you try double clicking?

This is what appears on the error log when double-clicking:
```
Uncaught TypeError: MutationObserver.observe: Argument 1 is not an object.
    <anonymous> chrome://conversations/content/stubWrapper.js:23
[stubWrapper.js:23:13](chrome://conversations/content/stubWrapper.js)
    <anonymous> chrome://conversations/content/stubWrapper.js:23
```
![image](https://user-images.githubusercontent.com/3778747/187211253-112121ad-8a0b-45af-808b-3a3b6501de8f.png)

> 
> Could there be another add-on install that is conflicting?

I tried disabling all other extensions and the error persist.

Thank you.

**Comment by alcarraz** [comment]
> What actually happened? Not a thing
>
> Just to confirm, does that mean no window opened at all.
>
Yes, it means exactly that, no window open, and no other visible response
from thunderbird. Except for the log line.

**Comment by bitchecker** [comment]
Same issue here!
Running Thunderbird 128.13.0esr (64 bit) on Fedora Linux 42, Thunderbird Conversations 4.3.1

I think that the problem is that when Thunderbird Conversations is installed, the double click action (that usually open the selected mail in a new window) is completely disabled, the selected mail is marked as read and the error is logged in error console.

If I temporarily disable the plugin, normal behaviour works and no errors in console, when I enable it, I get the reported error but, I also see too:
```
TypeError: subject.defaultView is null[window-global.sys.mjs:442:5](resource://devtools/server/connectors/js-process-actor/target-watchers/window-global.sys.mjs)
    observe resource://devtools/server/connectors/js-process-actor/target-watchers/window-global.sys.mjs:442
```

**Comment by Teagum** [comment]
I had this issue since yesterday. It appeared after I installed Thunderbird Conversation  4.3.9. This is the error displayed in the console right after double-clicking a message.

```
TypeError: can't access property "addEventListener", subject.defaultView is null[window-global.sys.mjs:463:5](resource://devtools/server/connectors/js-process-actor/target-watchers/window-global.sys.mjs)
    observe resource://devtools/server/connectors/js-process-actor/target-watchers/window-global.sys.mjs:463
```

Disabling Conversation restores the expected behaviour.
